src/hashesorg.py

```diff
@@ -0,0 +1,26 @@
+def HashesOrg(hashed, key):
+    try:
+        output = requests.get(f"https://hashes.org/api.php?key={key}&query={hashed}", timeout=5).text
+    except:
+        print("Couldn't connect")
+        return False
+
+    # Check for false positive
+
+    if "null" in output:
+        print("Hash not found") ; return False

```

Comment:
semi colons aren't super python, also logging file -- we don't need to tell the user that this one specific searcher failed -- they don't care :)